README.md

```diff
@@ -34,19 +36,19 @@ Next, create the initializer file by executing the Rails generator:
 
     $ bundle exec rails generate phraseapp_in_context_editor:install --access-token=<YOUR_TOKEN> --project-id=<YOUR_PROJECT_ID>
 
-##### --access-token
-
-You can create and manage access tokens in your [profile settings](https://app.phrase.com/settings/oauth_access_tokens) or via the [Authorizations API](https://developers.phrase.com/api/#authorizations).
-
 ##### --project-id
 
 You can find the ID of your project in your project settings in Translation Center.
 
+##### --account-id
+
+You can find the ID of your account in your account settings in Translation Center.
+
 ### Add the JavaScript helper
 
 Next, add the Javascript helper to your Rails application layout file:
 
-    <%= phraseapp_in_context_editor_js %>
+<%= phraseapp_in_context_editor_js %>
 
 If you don't want to use the helper but add the plain Javascript yourself, head over to our [documentation](https://help.phrase.com/help/translate-directly-on-your-website) to learn more.
```

Comment:
This link https://help.phrase.com/help/translate-directly-on-your-website that's used throughout the README goes to the CLI 🤔 

Maybe we should link to https://support.phrase.com/hc/en-us/articles/5784095916188-In-Context-Editor-Strings- instead?

The link for the In-Context Editor Demo under References section is also outdated.